crates/sui-framework/packages/sui-system/tests/staking_pool.move

```diff
@@ -167,6 +170,74 @@ module sui_system::staking_pool_tests {
         scenario.end();
     }
 
+    #[test]
+    fun test_convert_to_fungible_staked_sui_happy_e2e() {
+        let mut scenario = test_scenario::begin(@0x0);
+        set_up_sui_system_state(vector[@0x1]);
+
+        let mut system_state = scenario.take_shared<SuiSystemState>();
+        let staking_pool_id = system_state.validator_staking_pool_id(@0x1);
+
+        let staked_sui_1 = system_state.request_add_stake_non_entry(
+            coin::mint_for_testing(100 * 1_000_000_000, scenario.ctx()), 
+            @0x1, 
+            scenario.ctx()
+        );
+        test_scenario::return_shared(system_state);
+
+        advance_epoch_with_reward_amounts(0, 0, &mut scenario);
+
+        let mut system_state = scenario.take_shared<SuiSystemState>();
+        let latest_exchange_rate = system_state.pool_exchange_rates(&staking_pool_id).borrow(1);
+        assert!(latest_exchange_rate.sui_amount() == 200 * 1_000_000_000, 0);
```

Comment:
Super nit, here and everywhere: no need to add `, 0);` in assert macro. It can be used without an abort code.
```suggestion
        assert!(latest_exchange_rate.sui_amount() == 200 * 1_000_000_000);
```

Reply:
done